src/helpers/parseIpfs.ts

```diff
@@ -0,0 +1,58 @@
+import bs58 from 'bs58';
```

Comment:
why not use from js-utils?

Reply:
because js-utils package don't work with next js

Reply:
why not? probably a but there i should fix i guess

Reply:
ok, will double test and let you know

Reply:
ok, seems works with this webpack config `config.resolve.fallback = { fs: false, path: false };`